include/boost/histogram/utility/binomial_proportion_interval.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_UTILITY_BINOMIAL_PROPORTION_INTERVAL_HPP
+#define BOOST_HISTOGRAM_UTILITY_BINOMIAL_PROPORTION_INTERVAL_HPP
+
+#include <boost/histogram/detail/normal.hpp>
+#include <boost/throw_exception.hpp>
+#include <cmath>
+#include <stdexcept>
+
+namespace boost {
+namespace histogram {
+namespace utility {
+
+template <class ValueType>
+class binomial_proportion_interval {
+public:
```

Comment:
@gohil-jay Please add a `static_assert` here to constrain the `ValueType` to floating point numbers. You can use the template functions in the header `type_traits` from the standard library to check whether `ValueType` is a floating point type.